contrib/gitian-builder/gitian-build.sh

```diff
@@ -0,0 +1,414 @@
+#!/bin/bash
+
+# Copyright (c) 2016 The Bitcoin Core developers
+# Copyright (c) 2021 The Dogecoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+# GITIAN PROPERTIES
+unset USE_LXC
```

Comment:
Should we really add `USE_*` variables, docker will be specified in the `./bin/make-base-vm --docker --arch amd64 --suite trusty` command.

Do you care that variable are already set, having a conflict with the user ?